api/src/main/resources/org/openmrs/api/db/hibernate/PatientProgram.hbm.xml

```diff
@@ -74,6 +74,12 @@
 		<key column="patient_program_id" not-null="true"/>
 		<one-to-many class="org.openmrs.PatientState" />
 	</set>
+
+    <set name="attributes" lazy="true" inverse="true" batch-size="100"
+         cascade="all-delete-orphan" order-by="voided asc">
+        <key column="patient_program_id" />
+        <one-to-many class="org.openmrs.PatientProgramAttribute" />
```

Comment:
I think a package is defined at the top of the hbm file, so `PatientProgramAttribute` would be enough.